project/Cmd.scala

```diff
@@ -23,8 +23,13 @@ object Cmd {
   }
 
   def exec(cmd: Seq[String], wd: Option[File]): Result = {
+    val command =
+      sys.props.get("os.name").getOrElse("").toLowerCase match {
```

Comment:
Nice idea! Just a nitpicking comment: A more idiomatic way would be to use the destructuring feature of pattern  matching:
```scala
sys.props.get("os.name") match {
  case Some(name) if name.toLowerCase.startsWith("windows") => …
  case _ => …
}
```
(in this case you could also use `sys.props.getOrElse("os.name", "").toLowerCase` though :-))

or,  a more "functional" alternative:
```scala
sys.props.get("os.name")
    .map(_.toLowerCase)
    .filter(_.startsWith("windows"))
    .fold(cmd)(_ => Seq("cmd", "/C") ++ cmd)
```

Reply:
The functional alternative was also one of my candidates, but I had a mixed feeling about it. This is because the function returns a result from the parameters as a side effect without using the input value. But I'm not sure if that would be eligible or not. I haven't been around Scala long enough. But even with Java lambdas, I would have done without a functional notation, because it is easier to extend the case-match for other OS.

Anyway, the 

```
sys.props.get("os.name") match {
  case Some(name) if name.toLowerCase.startsWith("windows") => …
  case _ => …
}
```

looks great.

Reply:
I'm fine with any of these variants. I personally take the build tool area as being a kind of yolo place, tbh :-). So I wouldn't care too much, but I share the thoughts. I can just merge it as well, if you want.

Reply:
Honestly, I am very happy to learn the different ideas. And it helps me a lot to see what other possibilities there are. But since I'm on sbt-openapi-schema with Sys.scala, maybe there is again the consideration what works best.